source/guide_mailman.rst

```diff
@@ -0,0 +1,235 @@
+
+Step 1
+------
+
+Download the latest Mailman 2.1 version from https://ftp.gnu.org/gnu/mailman/ and extract the archive in your webroot
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/isabell
+ [isabell@stardust ~]$ wget https://ftp.gnu.org/gnu/mailman/mailman-99.9.9.tgz
+ [isabell@stardust ~]$ tar xzvf mailman-99.9.9.tgz
+ [isabell@stardust ~]$ cd mailman-99.9.9
+ [isabell@stardust ~]$
+
+Step 2
+------
+Create the installation folder
+
+::
+
+ [isabell@stardust ~]$ mkdir /var/www/virtual/isabell/mailman
+ [isabell@stardust ~]$ chmod a+rx,g+ws /var/www/virtual/isabell/mailman
+
+and run the configure script, telling Mailman where to install and what user/groups to use for its binaries:
+
+* ``--with-username``: Specify a different username than mailman. The value of this option can be an integer user id or a user name. 
+* ``--with-groupname``: Specify a different group than mailman. The value of this option can be an integer group id or a group name. 
+* ``--prefix``: Standard GNU configure option which changes the base directory that Mailman is installed into. By default $prefix is /usr/local/mailman
+* ``--with-mail-gid``: Specify an alternative group for running scripts via the mail wrapper.
+* ``--with-cgi-gid``: Specify an alternative group for running scripts via the CGI wrapper.
+
+::
+
+ [isabell@stardust ~]$ ./configure --with-username=isabell --with-groupname=isabell --prefix=/var/www/virtual/isabell/mailman/ --with-mail-gid=isabell --with-cgi-gid=isabell
+ [isabell@stardust ~]$
+
+After configuration is finished, you may compile and install the package by running
+
+::
+
+ [isabell@stardust ~]$ make && make install
+
+Step 3
+------
+
+If compilation and installation finished without errors, we can continue by checking folder permissions in the installation folder.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/isabell/mailman
+ [isabell@stardust ~]$ bin/check_perms
+
+In case errors are found, you should definitely fix them before continuing.
+
+Step 4
+------
+
+If you want the webinterface to be available publically, we need to create a couple of SymLinks and an htaccess-file.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/isabell/html
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/mailman/cgi-bin ./mailman
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/mailman/archives/public ./pipermail
+
+Create the file ``/var/www/virtual/isabell/mailman/cgi-bin/.htaccess`` with the following content:
+
+::
+
+ Options +ExecCGI
+ SetHandler cgi-script
+
+Finally, we need to adjust file permissions for the Mailman_ cgi-scripts to run:
+
+::
+
+ [isabell@stardust ~]$ chmod -R 0755 /var/www/virtual/isabell/mailman/cgi-bin
+
+Step 5
+------
+
+Because Mailman_ doesn't handle our .qmail-configuration automatically, we need to help it create the necessary aliases. The following script is based on the script provided in the official installation instructions and may be used by placing it in your Uberspace home-folder:
```

Comment:
I think ~/bin would be more suitable. It also needs to be made executable before it can be used